### DISCUSSION TITLE
When is a new update coming out?

### DISCUSSION THREAD

**Discussion by Josh65-2201** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#370] [General]
There have been many changes to the program since 1.0.0 rc2 and I cant get it to build. The last update was like 7 months ago. So I think there needs to be an official release some time soon.

**Comment by Melvin-Abraham** [comment]
I'm planning to release it probably this month soon... 🍿

**Reply by opticSquid** [reply]
Same question please make another release soon

**Comment by Antoni-Czaplicki** [comment]
Hi @Melvin-Abraham, when are you planning to release the new version?

**Reply by lucasdeeiroz** [reply]
Yeah, I'm not using it because the released version is eating a lot of RAM, and I am not able to build it for some reason. An update is really needed.

**Reply by Melvin-Abraham** [reply]
There's still some bits and pieces that needs to be added/fixed to deliver a stable experience across platforms. Once that is done, a new release would definitely be released which would be either this or the next month maximum.
